public/env.js

```diff
@@ -0,0 +1,7 @@
+// Denne filen blir overskrevet av server.js med variabler fra nais.yml
+window.__PAM_STILLINGSOK_URL__="http://localhost:8080/stillinger";
+window.__PAM_CONTEXT_PATH__="/stillinger";
+window.__PAM_AD_USER_API__="http://localhost:9017/aduser";
+window.__LOGIN_URL__="http://localhost:9017/aduser/local/cookie?subject=12345";
+window.__LOGOUT_URL__="http://localhost:9017/aduser/";
+console.warn('Loaded development version of env.js');
```

Comment:
Usikker på om dette er beste approach. Men det er gjort for å få dev-server + q og p til å virke med varfiabler fra nais.yml. 
Denne env.js vil være sånn i dev mode, men vil bli overskrevet av server.js i q og p med variabler fra nais.
Andre forslag?

Reply:
Jeg syns du burde droppe miljøvariabler i frontendkoden i det hele tatt, og heller bare ha endepunkter som redirecter i node-serveren + proxy.

Reply:
Ok, dette kan vi gjøre etterhvert